pyaptamer/utils/pdb_to_aaseq.py

```diff
@@ -0,0 +1,43 @@
+__author__ = "satvshr"
+__all__ = ["pdb_to_aaseq"]
+
+import os
+
+import pandas as pd
+from Bio import SeqIO
+
+
+def pdb_to_aaseq(pdb_file_path, return_df=False):
+    """
+    Extract amino-acid sequences (SEQRES) from a PDB file.
+
+    Parameters
+    ----------
+    pdb_file_path : str or os.PathLike
+        Path to a PDB file.
+    return_df : bool, optional, default=False
```

Comment:
let's call this `return_type` with two possible values `"pd.df"` and `"list"`. This makes it more upwards compatible.